### DISCUSSION TITLE
I'm new-ish to this  -  what's the difference between "sub-releases" (for lack of knowing the correct term)?

### DISCUSSION THREAD

**Discussion by xvdn13** [trapexit/mergerfs#1117] [Q&A] [answered]
I'm sorry if this has been asked, but I couldn't find any similar topics.  But as my title states, I'm new-ish to mergerfs.  I have used it successfully for a few months but am now wondering what the difference is between "sub-releases" (again, for lack of knowing the correct term)?  

For example, if I want to install the newest debian amd64 release, how do I know if I should choose "stretch," "buster," or "bullseye?"  Is there any difference between them?  Or can I just pick whatever name I like best (kidding - kind of)?  

Not urgent - I have mergerfs up and running currently.  I was just looking to update to the latest version and was curious.  Any help is appreciated.

**Comment by trapexit** [comment, marked as answer]
By "new-ish to this" you mean new to Linux I'm guessing?

The "sub-releases" are builds for different operating system releases. "stretch", "buster", etc. are the names of Debian releases. How do you know what to pick? Hopefully if you installed the OS you know what you chose. If not you can often use a tool like `lsb_release -a` to find out.

**Reply by xvdn13** [reply]
newish to linux; even newer to mergerfs.  this was the first install I had to choose a release for.  so thank you for your help - i appreciate it.  lsb_release was the tool I was looking for.  just slowly  learning my way around more every day.

edit: punctuation
